release/api/v1alpha1/eksarelease_types.go

```diff
@@ -0,0 +1,54 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// EKSARelease is the mapping between release semver of EKS-A and a Bundles resource on the cluster.
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+type EKSARelease struct {
```

Comment:
Do we not also need an `EKSAReleaseList` type?

Reply:
Yes, I added it.